sdk/translation/Azure.AI.Translation.Text/src/TextTranslationClient.cs

```diff
@@ -0,0 +1,1752 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Pipeline;
+using Azure.Core;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Threading;
+using System.Linq;
+using Azure.AI.Translation.Text.Models;
+using static System.Net.Mime.MediaTypeNames;
+using System.Text.Json;
+
+namespace Azure.AI.Translation.Text
+{
+    /// <summary> The Translator service client. </summary>
+    public partial class TextTranslationClient
```

Comment:
In general all custom files should go into a mirrored directory as their corresponding generated file.

The file that this is customizing is in `./Generated/TextTranslationClient.cs` so the custom code should go into `./Custom/TextTranslation.cs`.

This makes it very easy at a glance to see what is customized and know exactly where to find the corresponding generated file.

Reply:
OK, done.